application-confluence-migrator-pro-ui/src/main/resources/ConfluenceMigratorPro/Code/CommonCode.xml

```diff
@@ -178,6 +187,36 @@
       $('a[href$="#' + $(this).attr('id') + '"]').find('.cfmTitleIcon').removeClass('active');
     }
   });
+
+  $(function() {
+    // Since the associated liveData is refreshed on file upload, we don't need a response container.
+    new XWiki.FileUploader($('#AddAttachment input[type=file]')[0], {
+      'progressAutohide': true,
+      'responseContainer' : document.createElement('div')
+    });
+  });
+
+  $(document).on('xwiki:html5upload:done', function(e) {
+    $('#confluencePackages').data('liveData').updateEntries();
```

Comment:
You don't know if the event was triggered by the confluence package upload, so you may refresh the packages live data for nothing. It's not a big deal, but if there is an easy way to detect the source of the even then it would be nice to use it. Otherwise, at least add a comment.